esx_service/vmdk_ops.py

```diff
@@ -1535,11 +1547,13 @@ def execRequestThread(client_socket, cartel, request):
         group_info = vsi.get("/vm/%s/vmmGroupInfo" % vmm_leader)
         vm_name = group_info["displayName"]
         cfg_path = group_info["cfgPath"]
-        uuid = group_info["uuid"]
+        uuid = group_info["uuid"]            # BIOS UUID, see http://www.virtu-al.net/2015/12/04/a-quick-reference-of-vsphere-ids/
+        vc_uuid = group_info["vcUuid"]       # VC UUID
```

Comment:
if it's invalid then something else is fundamentally broken :-) as this is kernel info managing VM groups. IMO it's just too verbose to check for things which can only be out bugs in the areas double checked on every test ...